spec/CloudCode.spec.js

```diff
@@ -3494,9 +3494,17 @@ describe('afterLogin hook', () => {
       expect(req.object._context).not.toBeDefined();
       expect(req.object.context).not.toBeDefined();
     });
-    const obj = new TestObject();
-    obj.set('_context', { hello: 'world' });
-    await obj.save(null, { context: { a: 'a' } });
+    await request({
```

Comment:
Why does this require a HTTP request? Shouldn't the test also work using the JS SDK as it was?

Reply:
New SDK update now throws 

Error: Invalid key name: "_context"

REST is use here as a bypass

Reply:
I think a field name prefixed with underscore indicates an internal field of Parse Server. Internal fields can be accessed read/write using the `maintenanceKey` according to [access scope](https://github.com/parse-community/parse-server?tab=readme-ov-file#access-scopes). So I think the SDK should allow them to be sent to Parse Server. Parse Server then should check whether the `maintenanceKey` is set and allow the op. Does that make sense? Which PR introduced that restriction in the JS SDK?

Reply:
It’s been there since the beginning

Reply:
But why did the test pass with the previous version then?

Reply:
The error was silently handled

Reply:
Is that because of https://github.com/parse-community/Parse-SDK-JS/pull/2445, which we talked about some days back?

Reply:
Yes

Reply:
Ok, just want to make sure that is documented as a breaking change, so I think we're good here with the test change. I've opened https://github.com/parse-community/Parse-SDK-JS/issues/2474 to remove the prefix restriction.